runtime/manta/src/weights/pallet_tx_pause.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright 2020-2023 Manta Network.
+// This file is part of Manta.
+//
+// Manta is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Manta is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Manta.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Autogenerated weights for pallet_tx_pause
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2023-04-20, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("calamari-dev"), DB CACHE: 1024
+
+// Executed Command:
+// ./target/production/manta
+// benchmark
+// pallet
+// --chain=calamari-dev
```

Comment:
Am I wrong about the command to generate weight file for manta runtime? It should be something like `manta-dev`.
Or will you generate it by a release PR?

Reply:
Yeah I copied the weights from calamari...

This should be updated in release

Reply:
yeah we'll update it on the release PR